tests/tests_flex_array.cpp

```diff
@@ -34,7 +35,8 @@ TEST_SUITE("flex_array") {
 		ref.resize(expected_size);
 		std::iota(ref.begin(), ref.end(), 1);
 
-		CHECK_EQ(std::accumulate(f.begin(), f.end(), 0), std::accumulate(ref.begin(), ref.end(), 0));
+		CHECK(std::ranges::equal(f, ref));
+		CHECK(std::ranges::equal(f | std::views::reverse, ref | std::views::reverse));
```

Comment:
```suggestion
				CHECK(std::ranges::equal(std::ranges::subrange(f.rbegin(), f.rend()), ref | std::views::reverse));
```

Otherwise, I am  not very confident that the functions are actually called. Couldn't test it because of the boost build issue.